### PR TITLE
vochain: save process metadata

### PR DESCRIPTION
**Pull request by jordipainan**
- update proto version and change types accordingly

### CODE REVIEW

File: ethereum/handler/ethereumHandler.go [hunk 1]
```diff
@@ -223,6 +223,12 @@ func (eh *EthereumHandler) NewProcessTxArgs(ctx context.Context, pid [types.Proc
 		return nil, fmt.Errorf("error decoding entity address: %w", err)
 	}
 
+	// process metadata
+	if len(processMeta.MetadataCensusRootCensusUri[0]) == 0 {
```
Comment (p4u) [inline ethereum/handler/ethereumHandler.go]: I'd allow creating a process without metadata, so I'd remove this check.

File: oracle/oracle.go [hunk 2]
```diff
@@ -42,6 +42,9 @@ func (o *Oracle) NewProcess(process *models.Process) error {
 	if len(process.EntityId) != types.EntityIDsize {
 		return fmt.Errorf("entityId size is wrong")
 	}
+	if len(*process.Metadata) == 0 {
```
Comment (p4u) [inline oracle/oracle.go]: same

**Review by p4u — CHANGES_REQUESTED** [review]

File: oracle/oracle.go (hunk 2 above)
Comment (jordipainan) [reply oracle/oracle.go]: oki, thank you pau

**Comment by jordipainan** [comment]
@p4u  all solved

**Review by p4u — APPROVED** [review]

**Comment by p4u** [comment]
Great, thank you!

**Pull request merged**